ui/cypress/e2e/onboarding.test.ts

```diff
@@ -5,6 +5,23 @@ interface TestUser {
   bucket: string
 }
 
+describe('Onboarding Redirect', () => {
+  let user: TestUser
+  beforeEach(() => {
+    cy.flush()
+
+    cy.fixture('user').then(u => {
+      user = u
+    })
+
+    cy.visit('/')
+  })
+  it('Can redirect to onboarding page', () => {
```

Comment:
Can we add some space for these to breathe?
```suggestion

  it('Can redirect to onboarding page', () => {
```

Reply:
space added!